src/components/Pools/DetailModal/Header/styled.tsx

```diff
@@ -18,7 +18,7 @@ export const HeaderRoot = styled(Box)`
 
 export const StatsWrapper = styled(Box)`
   display: grid;
-  grid-template-columns: repeat(5, auto);
+  grid-template-columns: repeat(6, auto);
```

Comment:
does this grid needs to be 5 or 6 based on version ?

Reply:
Yes, because when I add "you apr" break the layout

Reply:
yes but what if its version 2 ? should it be `5` instead of `6` ?